CHANGELOG.md

```diff
@@ -212,6 +212,45 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [#1573](https://github.com/FuelLabs/fuel-core/pull/1573): Remove nested p2p request/response encoding. Only breaks p2p networking compatibility with older fuel-core versions, but is otherwise fully internal.
 
 
+## [Version 0.22.4]
```

Comment:
Marking this as available since `0.22.4` seems weird, since it's not availble on `0.23.0` at all IIUC.

Reply:
But it will be available in the `fuel-core 0.24`(= If someone ask for it in the `fuel-core 0.23`, we will patch it=)

Reply:
I like to be technically correct as well :) although I agreee that it doesn't matter much here.